test-proxy/pom.xml

```diff
@@ -134,6 +134,39 @@
             <skip>true</skip>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-shade-plugin</artifactId>
+          <version>3.2.4</version>
```

Comment:
Do we need the version here? I think it's defined in the bigtable-parent. Can you try build it without the version?

Reply:
> I also updated the README to remove the "build with old release"

But I think we should document it somewhere, and readme is a good place for this? 

> For the other plugins, there are warnings though (they are not introduced by this PR, we may deal with them in the future)

I think you can ignore those.

Reply:
Done. Thanks! I will squash the commits after it looks good.